<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 600 italic 20pt &quot;Segoe UI&quot;;
font: 18pt &quot;Segoe UI&quot;;
font: 16pt &quot;Segoe UI&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Capturar</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>CAPTURAR NEURONAS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>240</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Cerrar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCapturar">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>70</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Capturar</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>197</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtID">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Voltaje:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="txtVoltaje">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Posicion X:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="txtPx">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Posicion Y:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="txtPy">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Red:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="txtRed">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Green:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="txtGreen">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Blue:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="txtBlue">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Lista</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QListWidget" name="listWidget"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>497</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
